example/typescript/src/misskey/timeline.ts

```diff
@@ -1,20 +1,20 @@
-import generator, { MegalodonInterface, Entity, Response } from 'megalodon'
+import generator, { MegalodonInterface, Entity, Response, Converter } from '@cutls/megalodon'
 
 declare var process: {
   env: {
     MISSKEY_ACCESS_TOKEN: string
   }
 }
 
-const BASE_URL: string = 'https://misskey.io'
+const BASE_URL: string = 'https://chkukfsatpl3.cutls.dev'
 
-const access_token: string = process.env.MISSKEY_ACCESS_TOKEN
+const access_token: string = process.env.MISSKEY_ACCESS_TOKEN || '8zrWrxUIBNQ0kWUw'
```

Comment:
Is this a raw access token? Is it no problem?

Reply:
It's an example token